lib/newgistics/package.rb

```diff
@@ -0,0 +1,13 @@
+module Newgistics
+  class Package
+    include Newgistics::Model
+
+    attribute :tracking_number, String
+    attribute :weight, Float
+    attribute :billable_weight, Float
+    attribute :height, Float
+    attribute :width, Float
+    attribute :depth, Float
+    attribute :items, Array[Item]
```

Comment:
I'm wondering if we should create a new type of `Item` for this, like we did with `ManifestItem`, we could call it `PackageItem`. 

The main reason why I'd argue it's a good idea, is because the attributes of a package item will not always be the same as the attributes for an order item or a shipment item, what do you think @jrusso1020?

Reply:
PackageItem has been split into its own model with test coverage added.